borg/archive.py

```diff
@@ -571,10 +573,15 @@ def stat_attrs(self, st, path):
             b'mode': st.st_mode,
             b'uid': st.st_uid, b'user': uid2user(st.st_uid),
             b'gid': st.st_gid, b'group': gid2group(st.st_gid),
-            b'atime': int_to_bigint(st.st_atime_ns),
-            b'ctime': int_to_bigint(st.st_ctime_ns),
             b'mtime': int_to_bigint(st.st_mtime_ns),
         }
+        # borg can work with archives only having mtime (older attic archives do not have
+        # atime/ctime). it can be useful to omit atime/ctime, if they change without the
+        # file content changing - e.g. to get better metadata deduplication.
+        if not self.noatime:
```

Comment:
Normally I'd rather avoid the double negation, but since "noatime" is pracitcally a fixed/well-known word in unix-lands, let's stick with it.